team.html

```diff
@@ -385,15 +387,27 @@ <h5 class="font-weight-bold text-white mb-lg-4 pb-2 margin-top-mobile">Useful Li
                         </li>
                     </ul>
                 </div>
+
                 <div class="col-lg-3 col-md-3 col-sm-6 col-12">
                     <h5 class="font-weight-bold text-white mb-lg-4 pb-2 margin-top-mobile">Features</h5>
+
+                    <ul>
+                        <a class="text-white mb-lg-3" href="https://www.codechef.com/ide" target="_blank">Code Editor</a><br>
```

Comment:
Already few links are present and you added again just check and remove the duplicates.

Reply:
removed the duplicate links